pkg/virt-controller/services/template_test.go

```diff
@@ -1161,6 +1161,50 @@ var _ = Describe("Template", func() {
 					},
 				),
 			)
+			When("NetworkBindingPlugins feature enabled", func() {
+				var vmi *v1.VirtualMachineInstance
+
+				BeforeEach(func() {
+					vmi = &v1.VirtualMachineInstance{ObjectMeta: metav1.ObjectMeta{Name: "testvmi", Namespace: "default"}}
+					vmi.Spec.Networks = []v1.Network{
+						{Name: "default", NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}}},
+						{Name: "blue", NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "test1"}}},
+						{Name: "red", NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "other-namespace/test1"}}},
+					}
+					vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{
+						{Name: "default", Binding: &v1.PluginBinding{Name: "test-binding"}},
+						{Name: "blue"}, {Name: "red"}}
+				})
+
+				It("should add network binding plugin net-attach-def to multus annotation", func() {
```

Comment:
I would also add a test for a ns/name nad name.
And as Eddy said, the test fit more mutlus_annotations_test.go.

Reply:
Done